### ISSUE TITLE
warning configuration in bsconfig.json is not being applied

### ISSUE THREAD

**Issue by farukg** [reasonml-editor/reasonml-idea-plugin#261]
plugin version: 0.90.1-2020.1

Hi, 
i disabled some warnings in bsconfig.json, like so:
`"warnings": {
    "number": "-27-26-32-44-45",
    "error": "+8"
  }
`
-44 is disabling the warnings about shadowing operators ( see also https://github.com/MinimaHQ/re-classnames#warning-44)
It works as expected: when i run in a terminal or even in the bucklescript tool window, the warning does not appear.

But the warning inside the editor won't disappear. I think the warnings inside the editor is not using the configuration from the bsconfig.json. 

Screenshot of the warning:
![bet-frontend  ~-projects-work-bet-frontend  –  -lib-D10m-Component-Form-D10m_Component_Checkbox re_011](https://user-images.githubusercontent.com/1577842/87222054-e68d6100-c370-11ea-902b-51d6920c133e.png)

PS: Also, i just tested if the warning configured as an error ("+8") works and it doesn't either. It is still shown as a warning.

**Comment by giraud** [comment]
if you're using bs-platform >= 8.4, can you try with plugin 0.97 ?

**Comment by farukg** [comment]
I just did try with 8.4.2 and 0.97 as you asked for. The error/warning is not showing up anymore. 
I also tried changing bsconfig again to see if an error would appear, unfortunately it doesn't. No errors whatsoever, not even syntax errors. 
Anyways, for me personally, the current state is better ^^, since i can always use a separate terminal.  Please feel free to ask me for further tests or to close this issue. Thanks :)

**Comment by giraud** [comment]
I didn't start investigation, I was just hoping that it works-by-chance 😄

**Comment by giraud** [comment]
I tried your use case: 
- I added re-classnames to a project (bs-platform@8.4.2 / plugin 0.97)
- I used className=Cn.("a"+"b") in a component -> got the warning
- add "warnings": { "number": "-27-26-32-44-45", "error": "+8" } in bsconfig.json
- ! synchronise files (ctrl + alt + y) -> ninja build file is not read immediately because of intellij buffers 
- warning gone

if you change something in bsconfig.json it's best to synchronise all files because disk updates are not immediately propagated, the virtual files are cached in intellij.

it was in a source directory, maybe you are experiencing the problem in a dev directory ?

**Comment by farukg** [comment]
I just did reset idea to default settings, activated all bundled plugins & deactivated all downloaded & installed ReasonML Plugin only. Activated every Inspection there is. Tripple-Checked bsconfig. Also Syncing is not the issue. I did turn the buffer off and js files work fine. Only re files won't show anything
In the idea logs i get following warning when i open a re file:
```
WARN -       ReasonML.hints.rincewind - Failure: Sys_error("/tmp/BS_develop/SomeFIle.cmt: No such file or directory")
```
That could be hint...?

**Comment by farukg** [comment]
omg...finally...:D the  problem disappears when i change bsconfing package-specs "modules" to "commonjs". Only when set to es6 or es-global the hints disappear. Same on bs-platform 9.0.0

**Comment by giraud** [comment]
oh man, yes, I have a local fix for es6 that I forgot to publish.

**Comment by giraud** [comment]
also dev folders have a similar problem I think

**Comment by giraud** [comment]
should be fixed in 0.98
